packages/flutter_tools/templates/plugin/ios-swift.tmpl/projectName.podspec.tmpl

```diff
@@ -0,0 +1,26 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint {{projectName}}.podspec' to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = '{{projectName}}'
+  s.version          = '0.0.1'
+  s.summary          = '{{description}}'
+  s.description      = <<-DESC
+{{description}}
+                       DESC
+  s.homepage         = 'http://example.com'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'email@example.com' }
+  s.source           = { :path => '.' }
+  s.source_files = 'Classes/**/*'
+  s.dependency 'Flutter'
+  s.platform = :ios, '8.0'
+
+  # Flutter.framework does not contain a i386 slice. Only x86_64 simulators are supported.
+  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'VALID_ARCHS[sdk=iphonesimulator*]' => 'x86_64' }
+
+  swift_versions = ['4.0', '4.2', '5.0', '5.1']
+  swift_versions << Pod::Validator::DEFAULT_SWIFT_VERSION if Pod::Validator.const_defined? "DEFAULT_SWIFT_VERSION"
+  s.swift_versions = swift_versions
```

Comment:
There's disagreement about whether to make swift_versions exact based on the plugin author's tested Swift version, or whether to make it as permissible as possible (like this).  We have a meeting to make a decision one way or the other.